src/rules/no-identical-functions.ts

```diff
@@ -44,14 +57,12 @@ const rule: TSESLint.RuleModule<string, string[]> = {
       recommended: 'error',
       url: docsUrl(__filename),
     },
-    schema: [
-      {
-        enum: ['sonar-runtime'],
-      },
-    ],
+    schema: [OPTION_SCHEMA, OPTION_SCHEMA],
```

Comment:
Did you figure out why we need it twice ?

Reply:
yeah, each repetition is a choice between two options so that you can provide them in any order